src/base.jl

```diff
@@ -344,6 +344,39 @@ function add_forecasts!(sys::System, forecasts)
     _add_forecasts!(sys.forecasts, forecasts)
 end
 
+"""
+    add_forecast!(sys::System, forecasts)
```

Comment:
This is essentially a copy of add_forecasts!().  Do we need it?  The user can call add_forecasts!([forecast]) instead of add_forecast!(forecast).  The latter is a bit nicer.  If you want do want to keep it then I suggest changing add_forecasts so that it calls add_forecast and does not contain duplicate code.

Reply:
see 96572e8